airflow/bin/cli.py

```diff
@@ -1502,6 +1565,10 @@ class CLIFactory(object):
             'func': pool,
             'help': "CRUD operations on pools",
             "args": ('pool_set', 'pool_get', 'pool_delete'),
+        }, {
+            'func': batch_set_pools,
+            'help': "CRUD operations on a batch of pools",
```

Comment:
There should be some mention that relative ordering or the args is important

Reply:
Sure. Will add a help message